bindings/python/cstest_py/src/cstest_py/cstest.py

```diff
@@ -405,34 +404,16 @@ def run_tests(self):
         self.stats.print_evaluate()
 
 
-def get_repo_root() -> str | None:
-    res = sp.run(["git", "rev-parse", "--show-toplevel"], capture_output=True)
-    if res.stderr:
-        log.error("Could not get repository root directory.")
-        return None
-    return res.stdout.decode("utf8").strip()
-
-
 def parse_args() -> argparse.Namespace:
     parser = argparse.ArgumentParser(
         prog="Python CSTest",
-        description="Pyton binding cstest implementation.",
+        description="Python binding cstest implementation.",
+    )
+    parser.add_argument(
+        dest="search_dir",
+        help="Directory to search for .yaml test files.",
+        type=Path,
```

Comment:
I you remove the default entry, you need to make this one a `required=True`

Reply:
Positional args are required by default. That indeed the actual fix

Reply:
Also this is more clean I think. You install _cstest_py_ as a normal python package then you can run it from whatever you want by specifying the required arg that represents the folder of tests you want to execute. This would remove any implicit assertion, such us stay in git repo and use git cli to derive the tests folder.

Reply:
Yeah, it is a good change.